LeagueLib/Files/InibinReader.cs

```diff
@@ -101,64 +101,63 @@ private bool DeserializeSegment(int type, bool skipErrors = true)
                     values[i] = new InibinValue(type, Convert.ToInt32(bits[i]));
                 }
             }
-            else if (type == 6) // 3x byte values - Resource bar RGB color
+            else if (type == 6) // 3x byte values *0.1f
             {
                 byte[] bytes = new byte[3];
 
                 for (int i = 0; i < count; i++)
                 {
                     _reader.BaseStream.Read(bytes, 0, bytes.Length);
-                    values[i] = new InibinValue(type, BitConverter.ToUInt32(new byte[4] { 0, bytes[0], bytes[1], bytes[2] }, 0));
+                    values[i] = new InibinValue(type, string.Format("{0} {1:} {2}", bytes[0]/10.0f, bytes[1]/10.0f, bytes[2]/10.0f));
                 }
             }
-            else if (type == 7) // 3x float values ??????
+            else if (type == 7) // 3x float values(color) - order not confirmed
             {
-                if (!skipErrors)
-                    throw new Exception("Reading 12 byte values not yet supported");
-
-                //Console.WriteLine("Tried to read 12 byte values");
                 for (int i = 0; i < count; i++)
                 {
-                    // 4 + 4 + 4 = 12
-                    _reader.ReadInt32();
-                    _reader.ReadInt32();
-                    _reader.ReadInt32();
-                    values[i] = new InibinValue(type, "NotYetImplemented");
+                    float b=_reader.ReadSingle();
```

Comment:
`var` preferred over declaring the type
whitespace missing around `=` (applies for next two line as well)